File: alafia-web-application/src/app/model/dto/courseDto.ts
export class CourseDto {
  courseId: string;
  orderId: string;
  clientId: string;
  bookingId: string;
  dinnerTableId: string;
  restaurantId: string;

  constructor(courseId: string,
              orderId: string,
              clientId: string,
              bookingId: string,
              dinnerTableId: string,
              restaurantId: string) {
    this.courseId = courseId;
    this.orderId = orderId;
    this.clientId = clientId;
    this.bookingId = bookingId;
    this.dinnerTableId = dinnerTableId;
    this.restaurantId = restaurantId;
  }
}

File: alafia-server/src/main/java/es/alafia/server/model/dto/CourseDTO.java
package es.alafia.server.model.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CourseDTO {
    private String courseId;
    private String orderId;
    private String clientId;
    private String bookingId;
    private String dinnerTableId;
    private String restaurantId;
    private String urlVideo;
}
